### PR TITLE
Completed Array-2- Please review

### PR DESCRIPTION
**Pull request by het3008**

### CODE REVIEW

File: Solution_findApper.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*
+
+Time :  O(n)
```
Comment (shrivatsav) [inline Solution_findApper.java:3]: Please specify what `n` indicates.

**Review by shrivatsav — COMMENTED** [review]
Good Work!

**Comment by shrivatsav** [comment]
Good Work!

**Comment by het3008** [comment]
Thank You

**Pull request closed without merging**